symphony/samples/ankaios_provider/state.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: v0.1
+workloads:
+  symphony:
+    runtime: podman
+    agent: agent_A
+    controlInterfaceAccess:
+      allowRules:
+      - type: StateRule
+        operation: ReadWrite
+        filterMask:
+        - "*"        
+    restartPolicy: ALWAYS    
```

I would change it to never, because if the container fails you can see the logs easily with `ank logs symphony` without having the restarts in-between.
```suggestion
    restartPolicy: NEVER
```